dists/base/base-d7-modules.make

```diff
@@ -208,10 +211,40 @@ projects[zen][version] = "5"
 ; LIBRARIES
 projects[composer_manager][subdir] = "contrib"
 
+; CKEditor Library + plugins
+libraries[ckeditor][download][type] = "get"
+libraries[ckeditor][download][url] = "http://download.cksource.com/CKEditor/CKEditor/CKEditor%204.5.7/ckeditor_4.5.7_full.zip"
+libraries[ckeditor][directory_name] = "ckeditor"
+libraries[ckeditor][type] = "library"
+
+libraries[ckeditor-plugin-lineutils][download][type] = "get"
+libraries[ckeditor-plugin-lineutils][download][url] = "http://download.ckeditor.com/lineutils/releases/lineutils_4.5.7.zip"
+libraries[ckeditor-plugin-lineutils][directory_name] = "ckeditor/plugins/lineutils"
+libraries[ckeditor-plugin-lineutils][type] = "library"
+
+libraries[ckeditor-plugin-widget][download][type] = "get"
+libraries[ckeditor-plugin-widget][download][url] = "http://download.ckeditor.com/widget/releases/widget_4.5.7.zip"
+libraries[ckeditor-plugin-widget][directory_name] = "ckeditor/plugins/widget"
+libraries[ckeditor-plugin-widget][type] = "library"
+
+libraries[ckeditor-plugin-image2][download][type] = "get"
+libraries[ckeditor-plugin-image2][download][url] = "http://download.ckeditor.com/image2/releases/image2_4.5.7.zip"
+libraries[ckeditor-plugin-image2][directory_name] = "ckeditor/plugins/image2"
+libraries[ckeditor-plugin-image2][type] = "library"
+
+libraries[ckeditor-plugin-iframedialog][download][type] = "get"
+libraries[ckeditor-plugin-iframedialog][download][url] = "http://download.ckeditor.com/iframedialog/releases/iframedialog_4.5.7.zip"
+libraries[ckeditor-plugin-iframedialog][directory_name] = "ckeditor/plugins/iframedialog"
+libraries[ckeditor-plugin-iframedialog][type] = "library"
+
+libraries[ckeditor-plugin-placeholder][download][type] = "get"
+libraries[ckeditor-plugin-placeholder][download][url] = "http://download.ckeditor.com/placeholder/releases/placeholder_4.5.7.zip"
+libraries[ckeditor-plugin-placeholder][directory_name] = "ckeditor/plugins/placeholder"
+libraries[ckeditor-plugin-placeholder][type] = "library"
 
 ; Amazon S3 cURL API
 libraries[s3-php5-curl][download][type] = "get"
-libraries[s3-php5-curl][download][url] = "https://github.com/tpyo/amazon-s3-php-class/archive/v0.5.1.zip"
+libraries[s3-php5-curl][download][url] = "https://github.com/tpyo/amazon-s3-php-class/tarball/master"
```

Comment:
I had some issues at some point using the master tarball, why not use a release for libraries?